setup.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+with open("requirements.txt") as fh:
+    required = fh.read().splitlines()
+
+with open(os.path.join("ner4opt", "_version.py")) as fp:
+    exec(fp.read())
+
+setuptools.setup(
+    name="ner4opt",
+    description="NER4OPT Library",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    version=__version__,
+    author="",
+    url="",
+    packages=(["models"] + setuptools.find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"])),
+    include_package_data=True,
+    install_requires=required,
+    python_requires=">=3.9",
+    classifiers=[
+        "License :: OSI Approved :: MIT License",
```

Comment:
IF this is gonna be MIT License, let's add that license to the repo

Reply:
Yes, I will add an MIT license to the repo.